packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -17,13 +17,23 @@ const GuidanceWrapper = styled.div`
   height: 100%;
   position: absolute;
   background-color: ${GUIDANCE_BACKGROUND};
+  border: 1px solid transparent;
   color: ${C_WHITE};
+  @media screen and (-ms-high-contrast: active) {
+    background-color: transparent;
+  }
+`;
 
+const Content = styled.div`
```

Comment:
Have you considered putting these styles into GuidanceMessage to avoid further nesting and another wrapping div?

Reply:
Yep, I felt the CSS would be too finicky by just applying it to the message. I felt the message should be just the text as it is currently contained in `<strong>` and then a container `content` which would make it easier to set HCM fixes.